docs/guides/authoring/Howlite/cta/index.md

```diff
@@ -0,0 +1,36 @@
+# CTA & CTAs List Components
+
+## Usage
+CTA component renders a single link (An anchor html tag) with look & feel of a button or text link (controlled by style variants).
```

Comment:
An anchor -> an anchor (make lower case)

Reply:
(controlled by style variants) -> not sure if clear for regular reader

Reply:
lower case letter - changed. For style variants, I made it a link to CTA variants section